README.md

```diff
@@ -151,6 +151,7 @@ Here are the events you can listen for.
 * `halfOpen` - emitted when the breaker state changes to `halfOpen`
 * `fallback` - emitted when the breaker has a fallback function and executes it
 * `semaphore-locked` - emitted when the breaker is at capacity and cannot execute the request
+* `health-check-failed` - emitted when a user-supplied health check function returns a rejected promise
```

Comment:
I wonder if it also makes sense to have a Health check pass event.

I have no argument for or against :)